CMakeLists.txt

```diff
@@ -50,7 +53,7 @@ cotire(bald_engine)
 enable_testing()
 file(GLOB TEST_SRC_FILES ${PROJECT_SOURCE_DIR}/tests/*.cpp src/Log.cpp)
 
-add_executable(runUnitTests ${TEST_SRC_FILES})
+add_executable(runUnitTests ${TEST_SRC_FILES} tests/glfw3_test.cpp)
```

Comment:
Why? There's GLOB which should take care about all .cpp files in /tests/ folder.